src/app/appointments/appointments-view/appointments-view.component.html

```diff
@@ -0,0 +1,76 @@
+<div class="container" *ngIf="appointmentsOnCurrentDay">
+  <div class="view-modal">
+    <div class="view-modal__controls">
+      <div class="view-modal__controls-date-time">
+        <div class="view-modal__controls-date-time__date">
+          <fa-icon [icon]="calendarIcon"></fa-icon>
+          <div>{{ getDate() }}</div>
+        </div>
+        <div class="view-modal__controls-date-time__time">
+          <fa-icon [icon]="clockIcon"></fa-icon>
+          <div>
+            {{
+              appointmentsOnCurrentDay.startHour +
+                ":00 - " +
+                appointmentsOnCurrentDay.endHour +
+                ":00"
+            }}
+          </div>
+        </div>
+      </div>
+      <div class="view-modal__controls__arrows">
+        <fa-icon
+          *ngIf="isTherePrevAppointment()"
```

Comment:
Avoid calling functions from template, it will happen too often (except for click events).
`*ngIf="appointmentsOnCurrentDay.appointments[0].index > 0"`